executor/executor_linux.cc

```diff
@@ -35,7 +36,17 @@ const unsigned long KCOV_TRACE_CMP = 1;
 const int kInFd = 3;
 const int kOutFd = 4;
 
+// The non-null address that is used in syzkaller for other architectures
```

Comment:
Let's find an address that works for arm32. Would something like `0x1b9bc20000ull` work?

Reply:
That works. I hope the address also works on 64-bit. I have updated the code accordingly.